src/ProtectedRoomsSet.ts

```diff
@@ -362,13 +362,12 @@ export class ProtectedRoomsSet {
                     // ignore - assume no ACL
                 }
 
-                // We specifically use sendNotice to avoid having to escape HTML
-                await this.managementRoomOutput.logMessage(
-                    LogLevel.DEBUG,
-                    "ApplyAcl",
-                    `Applying ACL in ${roomId}`,
-                    roomId,
-                );
+                await this.client.sendMessage(this.managementRoomId, {
+                    msgtype: "m.text",
+                    body: `Applying ACL in ${roomId}.`,
+                    format: "org.matrix.custom.html",
+                    formatted_body: `Applying ACL in <span data-mx-spoiler>${roomId}</span>.`,
```

Comment:
we'll need to escape HTML in all these spoilers. There should already be a function for it, somewhere.